bindings/python/src/models.rs

```diff
@@ -181,7 +181,7 @@ impl PyModel {
     ///
     /// Returns:
     ///     :obj:`List[str]`: The list of saved files
-    #[pyo3(text_signature = "(self, folder, prefix)")]
+    #[pyo3(signature = (folder, prefix=None, name=None), text_signature = "(self, folder, prefix, name)")]
```

Comment:
since the name has been deprecated you can just remove it here instead!

Reply:
Removed `name` from `text_signature` and updated Python stubs